### PR TITLE
Fix ios fonts

### LINKED ISSUE #21: Fix ios font family error

### PR DESCRIPTION
**Pull request by biswanath-tewari**
Refactors and fixes the font assets linking on the ios native side.
Note: we need a replacement for Modak-Regular font file, as it is not valid under the Apple ecosystem.

### CODE REVIEW

**Comment by biswanath-tewari** [comment]
We need a replacement for the Modak-Regular font file. As this font is not valid in the Apple ecosystem. Validated using the font-book app.

File: ios/.xcode.env [hunk 1]
```diff
@@ -0,0 +1 @@
+export NODE_BINARY=$(command -v node)
```
Comment (shreya-mishra) [inline ios/.xcode.env:1]: why did we added this?

File: ios/CalendarMobile.xcodeproj/project.pbxproj
Comment (shreya-mishra) [inline ios/CalendarMobile.xcodeproj/project.pbxproj:1]: I beleive this pbx file is automatically changed due to package.json, isnt it ?

File: ios/CalendarMobile/Info.plist [hunk 2]
```diff
@@ -1,55 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
+<dict>
```
Comment (shreya-mishra) [inline ios/CalendarMobile/Info.plist:4]: please explain this dict change

**Comment by shreya-mishra** [comment]
> _ No description provided. _

please add title and description of the changes.

File: ios/.xcode.env (hunk 1 above)
Comment (biswanathTewari) [reply ios/.xcode.env:1]: it is an autogenerated code by Xcode.

File: ios/CalendarMobile.xcodeproj/project.pbxproj
Comment (biswanathTewari) [reply ios/CalendarMobile.xcodeproj/project.pbxproj:1]: it is an autogenerated code by Xcode. No changes have been made to the package.json

File: ios/CalendarMobile/Info.plist (hunk 2 above)
Comment (biswanathTewari) [reply ios/CalendarMobile/Info.plist:4]: Auto refactoring done by Prettier while appending UIAppFonts keys.

**Review by shreya-mishra — APPROVED** [review]

**Pull request merged**